pkg/sql/planner.go

```diff
@@ -651,6 +651,11 @@ const (
 	KVStringOptRequireValue   KVStringOptValidate = `value`
 )
 
+// KVStringCaseInsensitiveOpts options which supports case Insensitive value
+var KVStringCaseInsensitiveOpts = map[string]struct{}{
+	"format": {},
```

Comment:
nit: Make sure when populating this map, refer to the keys by their option constant values (`changefeedbase.OptFormat`) in this case rather than the raw string